cicd/42c-conf-examples/branches/42c-conf.yaml

```diff
@@ -0,0 +1,33 @@
+# This example shows how to configure failure conditions based on the branch name.
+#
+# When the CI/CD plugin starts, it creates an API collection in 42Crunch Platform
+# using the name of the repository and branch directly from your source control.
+#
+# If you run the plugin on multiple branches in the same repository,
+# the plugin creates a separate API collection for each branch.
+#
+# You can specify the plugin configuration individually for each branch in your repository.
+# You can also use wildcards for the branch names to apply the configuration to all
+# branches with matching names.
+
+# In the following example, the branch called "main" has been configured to fail
+# if the minimum data validation score is under 70, and any branch with a name matching
+# the wildcard pattern "feature-*" to fail if the data validation score is under 50.
+# 
+# All other branches which don't match "main" or "feature-*" will match pattern "*" and will 
```

Comment:
"All other branches which don't match "main" or "feature-*" will match pattern "*" and will 
# receive minimum data score of 60" --> "All other branches are caught by the patter "*" and are configured to fail if the data validation score is under 60"